La-Mula-App-main/src/main/kotlin/com/lamulaapp/repository/Order_StatusRepository.kt

```diff
@@ -0,0 +1,10 @@
+package com.lamulaapp.repository
+
+import com.lamulaapp.domain.Order_Status
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.stereotype.Repository
+import java.util.*
+
+@Repository
+interface Order_StatusRepository: JpaRepository<Order_Status, UUID> {

```

Comment:
Recuerda que estamos usando por estandar camelCase.

Realiza este cambio en todas las clases que sea necesario.

las variales inician con minuscula y las clases inician con mayúscula.

```suggestion
interface OrderStatusRepository: JpaRepository<Order_Status, UUID> {

```